src/views/homepage/components/featured-content-grid/featured-card.tsx

```diff
@@ -12,11 +12,14 @@ const FeaturedCard = ({ children, className, href, title }: $TSFixMe) => {
 			href={href}
 		>
 			<div>{children}</div>
-			<div className={s.learnMoreCta}>
-				<Text size={300} weight="medium">
-					Learn More
-				</Text>
-				<IconArrowRight24 />
+			<div className={s.learnMoreCtaContainer}>
+				<StandaloneLinkContents
+					className={s.learnMoreCta}
+					icon={<IconArrowRight16 />}
+					iconPosition="trailing"
+					size="medium"
+					text="Learn more"
+				/>
```

Comment:
🗒️ Switched to leveraging `StandaloneLinkContents` which I'd forgotten to do. Also made the element smaller as requested from Kelly in the doc.